TypeScript.py

```diff
@@ -74,6 +74,14 @@ def is_typescript_scope(view, scopeSel):
 
     return view.match_selector(location, scopeSel)
 
+def is_special_view(cur_view):
+    """
+    Determine if the current view is a special view (quick panel to be the most 
+    common ones). active_view can only be normal views, therefore the id shouldn't 
```

Comment:
This docstring comment should explain what a special view is. The second sentence should be a plain comment and explain why your logic works. For instance:

```Python
# 'active_view' can only be a normal view, therefore if its 'id' does not match
# that of the current view, there must be special view in focus.
```